docs/best-practices/continuous-integration.md

```diff
@@ -94,6 +94,7 @@ Most setups will need the following environment variables
 - `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD`: You need to provide an [application specific password](#application-specific-passwords) if you have 2-factor enabled and use _pilot_ or _deliver_ to upload a binary to App Store Connect
 - `FASTLANE_SESSION`: You need to provide a [pregenerated session via `fastlane spaceauth`](#spaceauth) if you have 2-factor authentication enabled and want to use any actions that communicates with App Store Connect.
 - `LANG` and `LC_ALL`: These set up the locale your shell and all the commands you execute run at. _fastlane_ needs these to be set to an UTF-8 locale to work correctly, for example `en_US.UTF-8`. Many CI systems come with a locale that is unset or set to ASCII by default, so make sure to double-check whether yours is set correctly.
+- `SPACESHIP_ONLY_ALLOW_INTERACTIVE_2FA`: Explicitly prevent running of 2FA in non-interactive mode.
```

Comment:
```suggestion
- `SPACESHIP_ONLY_ALLOW_INTERACTIVE_2FA`: Explicitly prevents being prompted for 2FA in non-interactive mode.
```

@kohtenko @ainame is this more clear and still make sense to what the env var does? I think it reads better

Reply:
Yeah it does but I come to think this env var is the workaround that we should ideally fix on the code side😂

Reply:
I thought the env var was added _as the fix_ 😅 ?